ui-framework-documentation/layers.md

```diff
@@ -0,0 +1,214 @@
+# UI Test Automation Framework Layers
+* [Test Module](#test-module)
+    * [Test Data usage in Test Module](#test-data-usage-in-test-module)
+    * [Access to os-autoinst testapi from Test Module](#access-to-os-autoinst-testapi-from-test-module)
+    * [Access to other framework layers from Test Module](#access-to-other-framework-layers-from-test-module)
+* [Controller](#controller)
+    * [Test Data usage in Controller](#test-data-usage-in-controller)
+    * [Access to os-autoinst testapi from Controller](#access-to-os-autoinst-testapi-from-controller)
+    * [Access to other framework layers from Controller](#access-to-other-framework-layers-from-controller)
+* [Page](#page)
+    * [Test Data usage in Page](#test-data-usage-in-page)
+  *   [Access to os-autoinst testapi from Page](#access-to-os-autoinst-testapi-from-page)
+  *   [Access to other framework layers from Page](#access-to-other-framework-layers-from-page)
+* [Validation Test Module](#validation-test-module)
+    * [Test Data usage in Validation Test Module](#test-data-usage-in-validation-test-module)
+    * [Access to os-autoinst testapi from Validation Test Module](#access-to-os-autoinst-testapi-from-validation-test-module)
+    * [Access to other framework layers from Validation Test Module](#access-to-other-framework-layers-from-validation-test-module)
+
+    
+## Test Module
+
+Test Module is a layer containing test case steps that need to be
+executed on the system under test.
```

Comment:
(SUT) and use it in the rest of the document.

Reply:
Thank you. I also considered this abbreviation. As it well-known for everyone it makes sense to shorten the occurrences. Will change it.

Reply:
Usually you should introduce it first and then use.

Reply:
Did not update it yet, but will do this soon.